### PR TITLE
Edit configuration (unistall app)

### PR DESCRIPTION
**Pull request by KacperLukasiewiczQA**
It has been added to the appium configuration to uninstall the application after each scenario.

### CODE REVIEW

File: src/test/java/com/intive/template/MobileDriverSetup.java [hunk 1]
```diff
@@ -54,6 +54,7 @@ public static void startDriver(String scenarioName) throws MalformedURLException
             String applicationLocation = String.format("%s/src/test/resources/application/%s", currentPath, "RetroBoard.apk");
             capabilities_mobile.put("app", applicationLocation);
             DesiredCapabilities caps = new DesiredCapabilities(capabilities_mobile);
+            caps.setCapability("fullReset","true");
```
Comment (MarekPawlowskiAuto) [inline src/test/java/com/intive/template/MobileDriverSetup.java:57]: Ok, but you can remove this configuration to "default" json :) In this way runner method will be cleaner and all(most off) of Appium configs will relate on jsons files

**Review by MarekPawlowskiAuto — APPROVED** [review]
lgtm :) Please on qa-auto also explain, what was changed, most of people will could not catch this, at first.

**Review by MarekPawlowskiQA — APPROVED** [review]

**Review by KlaudiaLapcionek — APPROVED** [review]
lgtm

**Comment by KacperLukasiewiczQA** [comment]
Description in jira has been updated.

**Pull request merged**